src/app/modules/timesheet/services/timesheet.service.spec.ts

```diff
@@ -103,3 +120,25 @@ describe('TimesheetService', () => {
     });
   });
 });
+
+describe('Exertenal URL shortening service:', () => {
+  let service: TimesheetService;
+  let httpClientSpy: { post: jasmine.Spy };
+
+  beforeEach(() => {
+    httpClientSpy = jasmine.createSpyObj('HttpClient', ['post']);
+    service = new TimesheetService(<any> httpClientSpy);
+  });
+
+  it ('shortenUrl() should return a shortened URL', () => {
+    const shortUrl = 'https://link.acrabadabra.com/8355d'
+    const url ='https://vimeo.com/256549521';
+
+    httpClientSpy.post.and.returnValue(of({shortUrl: shortUrl}));
```

Comment:
`httpClientSpy.post.and.returnValue(of({ shortUrl }));`
If the key and the value are the same it's better to write just the key ;)